e2e/tests/profile/profile-page.spec.ts

```diff
@@ -145,6 +145,8 @@ describe('Profile', () => {
     ]) {
       const originalValue = await element.getValue();
 
+      console.log(`missingness test '${originalValue}' -> '${testText}' -> (empty), for element: `, element.getXpath());
```

Comment:
nit: I normally use `logger.info` after `import { logger } from 'libs/logger';` b/c it provides timestamp . I try not to use console.log anymore.